src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml

```diff
@@ -106,7 +106,7 @@
                     Visible="False" 
                     ReadOnly="True" />
 
-    <StringProperty Name="TreatSpecificWarningsAsErrors" 
+    <StringProperty Name="WarningsAsErrors" 
```

Comment:
Yes we should. Tagging @emgarten again to make sure that the implementation does that.